### ISSUE TITLE
Windows (10). Doesn't show up properly with taskbar placements

### ISSUE THREAD

**Issue by thegreatrazz** [maxogden/menubar#80]
When the taskbar is not positioned to the bottom or the right, the popup show up in the opposite corner (as if the taskbar was in those positions). 

![Screenshot #1](https://cloud.githubusercontent.com/assets/7696711/10737856/707c46a4-7c79-11e5-8241-91b8a1e78f78.png)
![Screenshot #2](https://cloud.githubusercontent.com/assets/7696711/10737857/708018e2-7c79-11e5-8955-fe7be05f17ae.png)

I don't have enough experience with JavaScript and Node.js, so I can't offer a pull request to fix it.

**Comment by karissa** [comment]
Does windows 10 show up as 'win32'? If so, this line would explain that https://github.com/maxogden/menubar/blob/master/index.js#L18

**Comment by z-ax** [comment]
Okay, seems like a good thread to as one more thing: what's the best approach to show the window directly after launch, in appropriate position according to the Tray icon?
This one shows it in the middle of screen:
```
// main is MenuBar instance with preloadWindow: true
main.on('after-create-window', function () {
    main.window.show();
});
```

**Comment by amaurymartiny** [comment]
Fixed in [`menubar@6.0.4`](https://www.npmjs.com/package/menubar/v/6.0.4) 🚀
